assets/scripts/set-git-safe-directory.ps1

```diff
@@ -0,0 +1,51 @@
+
+#=========================
+# Set directory as safe
+#=========================
+
+$wslPath = '///wsl$/';
+$wslLocalPath = '///wsl.localhost/';
+
+for ( $i = 0; $i -lt $args.count; $i += 2 ) {
+    if ($args[$i] -eq "-e") {
+        $ENGINE_FOLDER = $args[$i + 1].Trim("'") 
+    }
+    elseif ($args[$i] -eq "-o") {
+        $OPS_FOLDER = $args[$i + 1].Trim("'") 
+    }
+    elseif ($args[$i] -eq "-d") {
+        $DISTRO = $args[$i + 1].Trim("'") 
+    }
+    elseif ($args[$i] -eq "-es") {
+        $IS_ENGINE_SAFE = $args[$i + 1].Trim("'") 
+    }
+    elseif ($args[$i] -eq "-os") {
+        $IS_OPS_SAFE = $args[$i + 1].Trim("'") 
+    }
+    else {
+        throw "Invalid argument passed"
+        exit 1
+    }
+}
+
+$localEnginePathCommand = "git config --global --add safe.directory '%(prefix)$wslPath$DISTRO$ENGINE_FOLDER'";
+$localhostEnginePathCommand = "git config --global --add safe.directory '%(prefix)$wslLocalPath$DISTRO$ENGINE_FOLDER'";
+
+$localOpsPathCommand = "git config --global --add safe.directory '%(prefix)$wslPath$DISTRO$OPS_FOLDER'";
+$localhostOpsPathCommand = "git config --global --add safe.directory '%(prefix)$wslLocalPath$DISTRO$OPS_FOLDER'";
+
+if ($IS_ENGINE_SAFE -eq 'False') {
```

Comment:
As mentioned after the variable pass is changed from string to boolean. These checks should be updated aswell.